angular-techDraft/src/app/my-nav/my-nav.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="ui fluid placeholder">
+  <div class="image header">
+    <div class="container">
+      <img class="ui mini circular image" src="https://yt3.ggpht.com/-5oE3xInEpSw/AAAAAAAAAAI/AAAAAAAAAAU/5Q6ic5I5378/s88-mo-c-c0xffffffff-rj-k-no/photo.jpg">
```

Comment:
Image should be a placeholder. It should not have an actual picture. See Milanote for visual comparison.